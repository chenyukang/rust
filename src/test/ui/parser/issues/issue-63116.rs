// fixed by #66361
// error-pattern: this file contains an unclosed delimiter
impl W <s(f;Y(;]
