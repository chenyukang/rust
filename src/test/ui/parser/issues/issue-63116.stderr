error: this file contains an unclosed delimiter
  --> $DIR/issue-63116.rs:3:18
   |
LL | impl W <s(f;Y(;]
   |          -       ^
   |          |
   |          unclosed delimiter

error: mismatched closing delimiter: `]`
  --> $DIR/issue-63116.rs:3:14
   |
LL | impl W <s(f;Y(;]
   |              ^ ^ mismatched closing delimiter
   |              |
   |              unclosed delimiter

error: aborting due to 2 previous errors

